docs/develop/convert-javascript-to-typescript.md

```diff
@@ -1,37 +1,26 @@
 ---
 title: Convert an Office Add-in project in Visual Studio to TypeScript
 description: Learn how to convert an Office Add-in project in Visual Studio to use TypeScript.
-ms.date: 07/18/2022
+ms.date: 02/23/2023
 ms.localizationpriority: medium
 ---
 
 # Convert an Office Add-in project in Visual Studio to TypeScript
 
 You can use the Office Add-in template in Visual Studio to create an add-in that uses JavaScript, and then convert that add-in project to TypeScript. This article describes this conversion process for an Excel add-in. You can use the same process to convert other types of Office Add-in projects from JavaScript to TypeScript in Visual Studio.
 
-> [!IMPORTANT]
-> This article describes the *minimal* steps necessary to ensure that, when you press F5, the code will be transpiled to JavaScript which is then sideloaded automatically into Office. However, the code isn't very "TypeScripty". For example, variables are declared with the `var` keyword instead of `let` or `const` and they aren't declared with a specified type. To take full advantage of the strong typing of TypeScript, consider making further changes to the code.
```

Comment:
Should we move this note to its own section at the end of the article?

Reply:
It's misleading because 'let' and 'const' are javascript. To truly make the files TypeScript you'd need to update the code to add types to the parameters (for example.) I could add one example at the end of how to add a type (to make it apparent none of the code is using TypeScript syntax and you might want to update it.)

Reply:
Good point. I'm okay omitting this then.